From 43d8a4970bbd908ed4b583335f1f4f42a3f6f354 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 23 Jan 2026 08:47:42 +0000
Subject: [PATCH] generator: Fix description of xfs_growfs

This function does not return "geometry information".  Remove
the bogus description.

(cherry picked from commit 6166304be685b2fafb8cc968a4558a906bb0609b)
---
 generator/actions_core.ml | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/generator/actions_core.ml b/generator/actions_core.ml
index 9c23da008..de95bdaf0 100644
--- a/generator/actions_core.ml
+++ b/generator/actions_core.ml
@@ -7332,11 +7332,7 @@ in the returned structure is defined by the API." };
          ["xfs_info"; "/"]], "ret->xfs_blocksize == 4096"), [];
     ];
     shortdesc = "expand an existing XFS filesystem";
-    longdesc = {|Grow the XFS filesystem mounted at C<path>.
-
-The returned struct contains geometry information.  Missing
-fields are returned as C<-1> (for numeric fields) or empty
-string.|} };
+    longdesc = {|Grow the XFS filesystem mounted at C<path>.|} };
 
   { defaults with
     name = "rsync"; added = (1, 19, 29);
-- 
2.47.3

